lib/db.js

```diff
@@ -342,6 +342,7 @@ class Database extends EventEmitter {
    * @see {@link https://mongodb.github.io/node-mongodb-native/4.0/interfaces/aggregateoptions.html#maxtimems}
    */
   async aggregate(collectionName, pipeline, options = {}) {
+    pipeline = deserialize(pipeline);
```

Comment:
curious about this change - is there a reason we weren't doing this for aggregations before now? peeking at the code it looks like we already do it for all the other querying operations in this file.

Reply:
I think it was just a miss, and should have been here the entire time